.circleci/config.yml

```diff
@@ -72,9 +72,13 @@ jobs:
       - run: curl --silent --show-error --location --fail --retry 3 --output /tmp/helm.tar.gz  https://storage.googleapis.com/kubernetes-helm/helm-v2.12.1-darwin-amd64.tar.gz &&
              tar -xz -C /tmp -f /tmp/helm.tar.gz &&
              mv /tmp/darwin-amd64/helm /usr/local/bin/helm
+      - run: go get -u gotest.tools/gotestsum
```

Comment:
this is potentially worth caching, but it was only 20s of a 5m32s run

(also, our existing homebrew caching looks like it might not be working, which makes me hesitant to dig into this)